recipe/meta.yaml

```diff
@@ -5,20 +5,19 @@
 {% else %}
   {% set name = "polars" %}
 {% endif %}
-{% set version = "0.20.31" %}
+{% set version = "1.0.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
-    sha256: 00f62dec6bf43a4e2a5db58b99bf0e79699fe761c80ae665868eaea5168f3bbb  # [name == "polars"]
-    sha256: 367633ef523e9ed717bc56f30a6d4bc5be1a5403bd4974bca12286f83fc36435  # [name == "polars-lts-cpu"]
-    sha256: 139794161c5f54ee8fabc6e4e0ee4f5ec4dad4f783000b0015f0b0b9c0b26879  # [name == "polars-u64-idx"]
+    sha256: 144a63d6d61dc5d675304673c4261ceccf4cfc75277431389d4afe9a5be0f70b  # [name == "polars"]
```

Comment:
SHAs are still missing for `polars-lts-cpu` and `polars-u64-idx` :smile:

Reply:
Will gladly add them, if you would be so kind and tell me where I can find them and the corresponding sources?

Reply:
Going through the Jinja template, you can simply build the URL yourself and then compute the SHA by downloading. For `polars-lts-cpu`, the URL would be:

```
https://pypi.io/packages/source/p/polars-lts-cpu/polars_lts_cpu-1.0.0.tar.gz
```

and you can compute the SHA via `curl`:

```
curl -L https://pypi.io/packages/source/p/polars-lts-cpu/polars_lts_cpu-1.0.0.tar.gz | sha256sum
```

Reply:
Thanks a lot.